web/src/ui/nodes/properties/execution-details.ts

```diff
@@ -48,21 +49,33 @@ export class UINodeExecutionDetails extends LitElement {
   @property({ type: Number })
   duration?: number
 
+  @property()
+  headerBg: string | undefined = undefined
+
   override render() {
     return html`
-      <stencila-ui-node-execution-state
-        status=${this.status}
-        required=${this.required}
-      ></stencila-ui-node-execution-state>
-      <stencila-ui-node-execution-count
-        value=${this.count}
-      ></stencila-ui-node-execution-count>
-      <stencila-ui-node-execution-ended
-        value=${this.ended}
-      ></stencila-ui-node-execution-ended>
-      <stencila-ui-node-execution-duration
-        value=${this.duration}
-      ></stencila-ui-node-execution-duration>
+      <stencila-collapsible-node-field
```

Comment:
Just noting that for consistency I renamed this to `stencila-ui-node-collapsible-property` in another PR, so there might have to be a renaming after that is merged in.